build.assets/robotest/run.sh

```diff
@@ -27,6 +27,7 @@ export GCE_VM=${GCE_VM:-custom-4-8192}
 # Parallelism & retry, tuned for GCE
 export PARALLEL_TESTS=${PARALLEL_TESTS:-4}
 export REPEAT_TESTS=${REPEAT_TESTS:-1}
+GRAVITY_PULL_OPTIONS=${GRAVITY_PULL_OPTIONS:-""}
```

Comment:
This doesn't appear to change anything for CI robotest runs -- I don't see a reference to this variable anywhere else in our Makefiles & toolchain. Is this for local testing?

Reply:
It is coming from [outside](https://github.com/gravitational/gravity.e/pull/4328/files#diff-b67911656ef5d18c4ae36cb6741b7965R30)